web/crux-ui/src/models/node.ts

```diff
@@ -66,6 +69,24 @@ export type NodeDeleteContainer = {
   prefix?: string
 }
 
+export type NodeAuditLogQuery = {
+  skip: number
```

Comment:
This very same schema is used for the auditlogs isn't it? Can we make a PaginationQuery type for this somewhere?